packages/stores/src/stores/CounterStore.ts

```diff
@@ -21,4 +22,8 @@ export default class CounterStore {
     decrement() {
         this.count = --this.count;
     }
+
+    unmount() {
+        stopPersisting(this);
```

Comment:
We should call `stopPersisting` here to solve the warning we get while running tests.